integration-tests/src/test/java/oracle/weblogic/kubernetes/ItFmwDiiSample.java

```diff
@@ -0,0 +1,403 @@
+
+    for (String param: params) {
+      String rcuSchemaPrefix = param.split(":")[1];
+
+      logger.info("Create RCU schema with fmwImage: {0}, rcuSchemaPrefix: {1}, dbUrl: {2}, "
+          + " dbNamespace: {3}:", FMWINFRA_IMAGE_TO_USE_IN_SPEC, rcuSchemaPrefix, dbUrl, dbNamespace);
+      assertDoesNotThrow(() -> createRcuSchema(FMWINFRA_IMAGE_TO_USE_IN_SPEC, rcuSchemaPrefix, dbUrl, dbNamespace),
+          String.format("Failed to create RCU schema for prefix %s in the namespace %s with dbUrl %s",
+              rcuSchemaPrefix, dbNamespace, dbUrl));
+    }
+
+
+
+    // create pull secrets for WebLogic image when running in non Kind Kubernetes cluster
+    // this secret is used only for non-kind cluster
+    createSecretForBaseImages(domainNamespace);
+
+    // install operator and verify its running in ready state
+    installAndVerifyOperator(opNamespace, domainNamespace);
+  }
+
+  /**
+   * Test JRF domain-in-image samples using domains created by wlst and wdt.
+   * Verify Pod is ready and service exists for both admin server and managed servers.
+   * Verify EM console is accessible.
+   *
+   * @param model domain name and script type to create domain. Acceptable values of format String:wlst|wdt
+   */
+  @ParameterizedTest
+  @MethodSource("paramProvider")
+  @DisplayName("Test FMW domain in image sample")
+  public void testFmwDomainInImage(String model) {
+
+    String domainUid = model.split(":")[1];
+    String script = model.split(":")[0];
+
+    setupSample();
+
+    //create WebLogic secrets for the domain
+    createSecretWithUsernamePassword(domainUid + "-weblogic-credentials", domainNamespace,
+            ADMIN_USERNAME_DEFAULT, ADMIN_PASSWORD_DEFAULT);
+    // create RCU credential secret
+    createRcuSecretWithUsernamePassword(domainUid + "-rcu-credentials", domainNamespace,
+        RCUSCHEMAUSERNAME, RCUSCHEMAPASSWORD, RCUSYSUSERNAME, RCUSYSPASSWORD);
+
+    Path sampleBase = Paths.get(tempSamplePath.toString(),
+        "scripts/create-fmw-infrastructure-domain/domain-home-in-image");
+
+    //update create-domain-inputs.yaml with the values from this test
+    updateDomainInputsFile(domainUid, sampleBase);
+
+    // change image name
+    assertDoesNotThrow(() -> {
+      replaceStringInFile(Paths.get(sampleBase.toString(), "create-domain-inputs.yaml").toString(),
+          "mode: wdt", "mode: " + script);
+      replaceStringInFile(Paths.get(sampleBase.toString(), "create-domain-inputs.yaml").toString(),
+              "domainHomeImageBase: container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.4",
+              "domainHomeImageBase: " + FMWINFRA_IMAGE_TO_USE_IN_SPEC);
+    });
+
+    // run create-domain.sh to create domain.yaml file, run kubectl to create the domain and verify
+    //verify EM console is accessible
+    createDomainAndVerify(domainUid, sampleBase);
+
+  }
+
+  private void createDomainAndVerify(String domainName, Path sampleBase) {
```

Comment:
use the method  oracle.weblogic.kubernetes.utils.CommonTestUtils.createDomainAndVerify